### DISCUSSION TITLE
Future development -> Experiment phase/Data Mining & Discovery??

### DISCUSSION THREAD

**Discussion by george0st** [mlrun/mlrun#1907] [Q&A] [answered]
Hi @yaronha, @adiso75, do you plan to focus on experiment phase also (or it is out of scope for future development)? I think about these topics e.g.:

- Data normalization (generate proposals, how to normalize data)
- Data patterns (identification possible patterns e.g. linear values, periodical values, group of values, etc.)
- Predictors discovery/definition (descovery-define proposal for possible predictors)
- etc.

**Comment by adiso75** [comment, marked as answer]
Hi @george0st  - as you know enhancing of the experiment phase with data mining features is not at the core of our product , however, we always look for integration with 3rd party tools or libraries that can help and complement our offering.

**Reply by george0st** [reply]
Thanks @adiso75
